Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Valid CCARCH options: aarch64 or the default x86_64 
+ARG CCARCH="x86_64"
+
+FROM --platform=$BUILDPLATFORM messense/rust-musl-cross:${CCARCH}-musl AS builder
+WORKDIR /home/rust/src
+COPY . .
+
+RUN rm rust-toolchain
+
+RUN cargo build --release
+
+RUN apt-get update && apt-get --only-upgrade install -y ca-certificates
+RUN update-ca-certificates
+
+RUN adduser \
+    --disabled-password \
+    --gecos "oracle-core" \
+    --home "/data" \
+    --shell "/bin/sh" \
+    --no-create-home \
+    --uid "9010" \
+    oracle-core
+
+FROM --platform=$TARGETPLATFORM busybox:stable-musl AS final
+ARG CCARCH="x86_64"
```

Comment:
Seems like `CCARCH` is already set in the beginning of this file.

Reply:
As far as I understand, every "FROM" is a different scope, and the initial CCARCH ARG would not make it here.

Reply:
Oh, I'm not that verse in docker. Good to know!